portal-api/src/v1/validation/fields/issued-date.js

```diff
@@ -22,45 +19,43 @@ module.exports = (facility, errorList, deal) => {
     manualInclusionNoticeSubmissionDate: minSubmissionDateTimestamp,
   } = deal.details;
 
-  const dealSubmissionDate = formattedTimestamp(dealSubmissionDateTimestamp);
-  const minSubmissionDate = formattedTimestamp(minSubmissionDateTimestamp);
-  const issuedDate = formattedTimestamp(facility.issuedDate);
-  const today = moment();
+  const dealSubmissionDate = new Date(Number.parseInt(dealSubmissionDateTimestamp, 10));
+  const minSubmissionDate = new Date(Number.parseInt(minSubmissionDateTimestamp, 10));
+  const issuedDate = new Date(Number.parseInt(facility.issuedDate, 10));
+  const today = new Date();
 
-  let dateOfSubmission = moment().utc().valueOf();
+  let dateOfSubmission = today.valueOf();
 
   if (dealSubmissionDateTimestamp || minSubmissionDateTimestamp) {
     if (submissionType === CONSTANTS.DEAL.SUBMISSION_TYPE.AIN) {
-      dateOfSubmission = dealSubmissionDate;
+      dateOfSubmission = new Date(dealSubmissionDate.valueOf());
     } else if (minSubmissionDateTimestamp) {
-      dateOfSubmission = minSubmissionDate;
+      dateOfSubmission = new Date(minSubmissionDate.valueOf);
     }
   }
 
-  if (dateHasAllValues(issuedDateDay, issuedDateMonth, issuedDateYear)) {
-    if (!moment(issuedDate).isSameOrAfter(dateOfSubmission, 'day')) {
-      const formattedDealSubmissionDate = moment(dateOfSubmission).format('Do MMMM YYYY');
+  const issuedDayHasValues = hasValue(issuedDateDay) && hasValue(issuedDateMonth) && hasValue(issuedDateYear);
+
+  if (issuedDayHasValues) {
+    if (isBefore(issuedDate, startOfDay(dateOfSubmission))) {
```

Comment:
As per `supply-contract-conversion-rate-date-is-required-for-non-GBP.js` (what a mouthful!), if we're only caring about dates, lets work as `startOfDay` throughout please for simplicity -- what do you think?

Also same comments on this file as `supply-contract-conversion-rate-date-is-required-for-non-GBP.js`